subprojects/docs/src/docs/userguide/custom_tasks.adoc

```diff
@@ -132,17 +132,41 @@ include::{samplesPath}/customPlugin/groovy/plugin/src/test/groovy/org/gradle/Gre
 [[incremental_tasks]]
 == Incremental tasks
 
-With Gradle, it's very simple to implement a task that is skipped when all of its inputs and outputs are up to date (see <<more_about_tasks.adoc#sec:up_to_date_checks,Incremental Builds>>). However, there are times when only a few input files have changed since the last execution, and you'd like to avoid reprocessing all of the unchanged inputs. This can be particularly useful for a transformer task, that converts input files to output files on a 1:1 basis.
+With Gradle, it's very simple to implement a task that is skipped when all of its inputs and outputs are up to date (see <<more_about_tasks.adoc#sec:up_to_date_checks,Incremental Builds>>).
+However, there are times when only a few input files have changed since the last execution, and you'd like to avoid reprocessing all of the unchanged inputs.
+This can be particularly useful for a transformer task, that converts input files to output files on a 1:1 basis.
```

Comment:
No comma. "...for a transformer task that converts..."

Reply:
Removed the comma in https://github.com/gradle/gradle/pull/8952.